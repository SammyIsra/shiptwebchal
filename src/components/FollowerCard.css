.FollowerCard {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start
}

.FollowerCard .avatar{
  width: 100%;
  background: white;
}

.FollowerCard {

  margin-bottom: 5px;
  background-color: rgba(225, 225, 225, 1);
}

.FollowerCard .user-label {
  margin-bottom: 5px;
  font-size: 2rem;

}