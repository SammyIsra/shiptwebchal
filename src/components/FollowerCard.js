import React from 'react';

import './FollowerCard.css';

function FollowerCard({avatar_url, login}){
  return (
    <div className="FollowerCard" >
      <img 
        className="avatar" 
        src={avatar_url} 
        alt={`Profile avatar of ${login}`} />
      <p className="user-label" >{login}</p>
    </div>
  )
}

export default FollowerCard;