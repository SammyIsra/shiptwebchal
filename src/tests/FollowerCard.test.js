import React from 'react';
import renderer from 'react-test-renderer';

import FollowerCard from '../components/FollowerCard';

test('renders a user card given valid user data', ()=>{

  const followerCard = renderer.create(
    <FollowerCard login="SammyIsra" avatar_url="http://i.imgur.com/KB4JPdv.png" />
  );

  expect(followerCard.toJSON()).toMatchSnapshot();
});