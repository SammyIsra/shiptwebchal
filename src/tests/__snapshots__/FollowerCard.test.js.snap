// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a user card given valid user data 1`] = `
<div
  className="FollowerCard"
>
  <img
    alt="Profile avatar of SammyIsra"
    className="avatar"
    src="http://i.imgur.com/KB4JPdv.png"
  />
  <p
    className="user-label"
  >
    SammyIsra
  </p>
</div>
`;
